README.md

```diff
@@ -15,6 +16,16 @@ As mentioned above, the package provides two things:
 - The CLI: a `dcictl` command is provided. For more details `dcictl --help`.
 - The API: a python module one can use to interact with a control server (`dciclient.v1.api.*`)
 
+### (experimental) On MacOS directly from github:
+
+1. Create a `venv`, run: `$ python3 -m venv myvenv`
+    - alternatively you may use other tools: `virtualenv`, `pipenv`, etc.
+1. Activate `myvenv`: `$ source myvenv/bin/activate`
+1. Install the package: `(myvenv) $ pip install git+https://github.com/redhat-cip/python-dciclient.git@3.1.0`
```

Comment:
As you saw in #16 you can't install directly from the github ref.

Instead you should rely on pypi and install the `dciclient` package from there: https://pypi.org/project/dciclient/

```
pip install dciclient
```